src/js/_enqueues/wp/customize/controls.js

```diff
@@ -699,6 +699,18 @@
 		construct = this;
 		params = params || {};
 		focus = function () {
+			// If a child section is currently expanded, collapse it.
+			if ( construct.extended( api.Panel ) ) {
+				var sections = construct.sections();
```

Comment:
Any reason to prefer declaring this `var` here or at the top of the function with the other four variables? I'm not sure either way.

Reply:
Originally, I thought it may be better to only declare `sections` at the point where it would be required.

However, since others like `completeCallback` are defined with the others despite only being used in certain conditions, I've moved `sections` up with the others for consistency. Pushing the changes now. Nicely spotted!